src/list.scss

```diff
@@ -573,21 +637,277 @@ $form-item: #{$fd-namespace}-form-item;
       .#{$block}__item:last-child {
         padding-bottom: $fd-list-byline-item-padding-y;
       }
+    }
+  }
+
+  // NAVIGATION INDICATION
+  &--navigation-indication {
+
+    .#{$block}__item {
+      &::after {
+        min-width: $fd-list-navigation-indicator-width - 1rem;
+        content: '';
+      }
+    }
```

Comment:
Is there any reason, why we have it ?

Reply:
<img width="1315" alt="Screen Shot 2020-04-28 at 8 35 28 AM" src="https://user-images.githubusercontent.com/39598672/80487741-47d59480-892b-11ea-9dab-053293287925.png">

Actually the answer to your 2 questions is one. Take a look at `List Item 4` and `List Item 5`. The elements that are on the right side should always be aligned, with or without navigation indicator. How I achieved is by using pseudo element that has a min-width which is the width of the nav indication. This space is now reserved for nav indication and the content to the left is aligned. If you have the class indicating that this element has nav indication you change the content of the pseudo element to >.